src/components/member/MembersOccasion/OccasionsCountRow/OccasionsCountRow.jsx

```diff
@@ -14,9 +14,9 @@ import { TIME_PERIOD } from '../../../../constants/common/commonConstants';
  */
 const OccasionsCountRow = props => (
   <Container className="occasions-count-row">
-    <OccasionCount count={props.todayCount} timePeriod={TIME_PERIOD.TODAY} />
-    <OccasionCount count={props.weekCount} timePeriod={TIME_PERIOD.WEEK} />
-    <OccasionCount count={props.monthCount} timePeriod={TIME_PERIOD.MONTH} />
+    <DashboardCalendarCard count={props.todayCount} name={TIME_PERIOD.TODAY} />
```

Comment:
timePeriod should be the correct prop name here.
```suggestion
    <DashboardCalendarCard count={props.todayCount} timePeriod={TIME_PERIOD.TODAY} />
```

Reply:
done